Formula/tapi.rb

```diff
@@ -0,0 +1,52 @@
+class Tapi < Formula
+  desc "Apple TAPI library (tpoechtrager fork)"
+  homepage "https://opensource.apple.com/tarballs/tapi"
+  url "https://github.com/tpoechtrager/apple-libtapi/archive/1100.0.11.tar.gz"
+  sha256 "bef360e713852f451383a7f1fdd4abfd17ee2d937778cd757708e52595d77b78"
+  license "APSL-2.0"
+
+  depends_on "cmake" => :build
+  depends_on "make" => :build
+
+  patch :p0, :DATA
+
+  def install
+    ENV.cxx11
```

Comment:
This shouldn’t be required, I’m guessing you based this of the Macports Portfile I’d done a while ago. In that we set that for the compiler standard to ensure a new enough compiler would be use for ancient systems.

Reply:
then removing this should be okay

Reply:
I believe it should be needed that setting should be handled within CMakeLists.txt so nothing for us to worry about.

Reply:
*Should not